.github/workflows/changelogs.yml

```
@@ -56,11 +56,11 @@ jobs:
       with:
         filter: '*.changes'
     - name: Fail if the master changelog files are modified
-      if: steps.master.outputs.all
+      if: steps.master.outputs.modified
```

just a doubt here...
I was looking at the possible states (https://github.com/Ana06/get-changed-files/tree/v2.2.0) and I'm  wondering if `added_modified_renamed` could  do  the job better than `modified`.

```suggestion
      if: steps.master.outputs.added_modified_renamed
```

---

Isn't that one aggregating the added, the modified and the renamed files? FWICT we want to avoid checking the added `.changes` files, and probably the renamed too